### PR TITLE
Update difference between a constructor and a method

### PR DESCRIPTION
**Pull request by jayesh83**
Provided an example with a syntax to understand better

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -509,9 +509,39 @@
     [encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)).
 
 * **What is the difference between a constructor and a method?** 
-    - The name of the constructor is same as that of the class name, whereas the name of the method can be anything.
-    - There is no return type of a constructor.
-    - When you make an object of a class, then the constructor of that class will be called automatically. But for methods, we need to call it explicitely.
```
Comment (mishra3452) [inline README.md:514]: We can keep these lines.

File: README.md [hunk 2]
```diff
@@ -509,9 +509,39 @@
...
+              
+                public void someMethod(){
+                  ...
+                }
+                // call method
+                someMethod()
+              }
+              
+    - Name of the method can be anything.
+    - When you make an object of a class, then the constructor of that class will be called automatically. But for methods, we need to         call it explicitely.
```
Comment (mishra3452) [inline README.md:544]: Do a little bit of formatting:
1. please formate the changes such as use bullet points and not 1, 2, 3....
2. remove extra space from the constructor and function snippet

Please preview the file after changing the text..

Comment (jayesh83) [reply README.md:544]: Okay formatting it

**Comment by mishra3452** [comment]
> Provided an example with a syntax to understand better

- Topic  
  - Sub-Topic
    - again sub-topic

use the above format and you can remove 1, 2, 3,.....numbering from it

File: README.md (hunk 1 above)
Comment (jayesh83) [reply README.md:514]: Yeah sure but don't I've already covered those points intuitively with example, should I now consider adding these again?

Comment (mishra3452) [reply README.md:514]: Keep the above 3 points as it is and if the same is present in the latter half of the text, then you can remove that...

File: README.md [hunk 3]
```diff
@@ -1330,4 +1354,4 @@
 ```
 
 ### Contributing to Android Interview Questions
-Just make pull request. You are in!
\ No newline at end of file
+Just make pull request. You are in!
```
Comment (mishra3452) [inline README.md:1357]: Also, keep this as it is

File: README.md [hunk 4]
```diff
@@ -508,22 +508,25 @@
...
-      though we can call parent class constructor using 'super' keyword.
-    - We call method directly using their name.
+    - We can call method directly using their name.
     - Constructor Syntax -         
               public class SomeClass{
                 SomeClass(parameter_list){ 
                   ...
                 } 
               ...
               }
```
Comment (mishra3452) [inline README.md]: Use code formatting for adding codes...

File: README.md [hunk 5]
```diff
@@ -508,22 +508,25 @@
...
-Note: 
-Constructor name is the same as class name
-Constructor does not have return type
-
+    - Note: 
+        Constructor name is the same as class name
+        Constructor does not have return type
+
```
Comment (mishra3452) [inline README.md:529]: use this for **note**:
| Note: |
| :--- |
| In the above syntax, the name of the constructor is the same at that of class and it has no return type. |

**Comment by jayesh83** [comment]
@mishra3452 check out the latest 2 commits

File: README.md [hunk 6]
```diff
@@ -508,12 +508,36 @@
...
+    - We can call method directly using their name.
+    - Constructor Syntax -
+      ```java
+              public class SomeClass{
+                SomeClass(parameter_list){ 
+                  ...
+                } 
+              ...
+              }
+      ```
```
Comment (mishra3452) [inline README.md:527]: Change this to:
```java
public class SomeClass{
    SomeClass(parameter_list){ 
        ...
    } 
    ...
}
```
I mean just remove extra space from the front of each line....

File: README.md [hunk 7]
```diff
@@ -508,12 +508,36 @@
...
+        
+    - Method Syntax 
+      ```java public class SomeClass{
+                public void someMethod(parameter_list){
+                  ...
+                }
+                // call method
+                someMethod(parameter_list)
+              }
+      ```
```
Comment (mishra3452) [inline README.md:540]: Change this to(remove extra space):
```java 
public class SomeClass{
    public void someMethod(parameter_list){
        ...
    }
    // call method
    someMethod(parameter_list)
}
```

File: README.md [hunk 8]
```diff
@@ -508,12 +508,33 @@
...
-
+    - Constructor and a method they both run a block of code but the difference is in calling them.
+    - We can call method directly using their name.
+    - Constructor Syntax -         
+              public class SomeClass{
+                SomeClass(parameter_list){ 
+                  ...
+                } 
+              ...
+              }
```
Comment (mishra3452) [inline README.md:526]: Just remove extra space from the starting of every line i.e. remove extra space before `public` and the same goes with other lines also.... Change the class name to `SomeClassName`
Same is for the method part also..

Comment (jayesh83) [reply README.md]: Done!

File: README.md (hunk 7 above)
Comment (jayesh83) [reply README.md]: Done

File: README.md [hunk 9]
```diff
@@ -508,12 +508,37 @@
...
-
+    - Constructor and a method they both run a block of code but the difference is in calling them.
+    - We can call method directly using their name.
+    - Constructor Syntax -
+        ```java
+        public class SomeClassName{
+            SomeClassName(parameter_list){ 
+                ...
+            } 
+        ...
```
Comment (mishra3452) [inline README.md:525]: keep this line 4 space ahead... I mean it should appear just below the `SomeClassName(parameter_list)`

Comment (mishra3452) [inline README.md:525]: keep this line 4 space ahead i.e. it should lie just below `{`

File: README.md [hunk 10]
```diff
@@ -508,12 +508,37 @@
...
+        and it has no return type.
+        
+    - Method Syntax 
+        ```java
+        public class SomeClassName{
+            public void someMethodName(parameter_list){
+                ...
+            }
+        // call method
+        someMethodName(parameter_list)
```
Comment (mishra3452) [inline README.md:539]: keep these two lines 4 space ahead i.e. it should lie just below `}`

File: README.md (hunk 9 above)
Comment (jayesh83) [reply README.md]: done! is it okay to merge now?

**Pull request merged**